app/src/main/java/hm/orz/chaos114/android/androidsaddlersample/MainActivity.java

```diff
@@ -7,10 +7,16 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    // checkstyle error
+    public String HOGE;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        // findbugs error
+        HOGE.getBytes();
```

Comment:
: [I18N] Found reliance on default encoding in hm.orz.chaos114.android.androidsaddlersample.MainActivity.onCreate(Bundle): String.getBytes()